mirage-http-CHANGES.md

```diff
@@ -0,0 +1,73 @@
+### 3.1.0
```

Comment:
@rgrinberg Do you know if that file will be picked automatically by topkg for the releases? By default it picks CHANGES.md but maybe topkg-jbuilder does something clever? /cc @diml and @dbuenzli